PowerEditor/src/NppCommands.cpp

```diff
@@ -3746,14 +3746,26 @@ void Notepad_plus::command(int id)
 		{
 			NppParameters& nppParams = NppParameters::getInstance();
 
-			COLORREF colour{ nppParams.getCurLineHilitingColour() };
-			bool hilite{ nppParams.getSVP()._currentLineHilitingShow };
+			const COLORREF colour{ nppParams.getCurLineHilitingColour() };
+			const bool hilite{ nppParams.getSVP()._currentLineHilitingShow };

```

Comment:
Please add a white space before `{` for the sake of visibility.
```
const COLORREF colour { nppParams.getCurLineHilitingColour() };
const bool hilite { nppParams.getSVP()._currentLineHilitingShow };
```

Reply:
Done. (It was the VS auto-formatting at work.)